test/jdk/com/sun/crypto/provider/KDF/HKDFSaltIKMTest.java

```diff
@@ -26,7 +26,6 @@
  * @bug 8331008
```

Comment:
Should we add 8353888 to all the tests?

Reply:
The Virtual Threads JEP didn't. https://github.com/openjdk/jdk/commit/2586f36120317cd206464b1e79d3906f711487cb#diff-35150e3bfd3304405ec403aa29cbea7aaa05c7bab3e28ac92e8162683114746a. I assume we don't either.